pyproject.toml

```diff
@@ -58,6 +58,7 @@ dependencies = [
     "jiter==0.10.0",
     "joblib==1.5.1",
     "kiwisolver==1.4.9",
+    "locust>=2.42.0",
```

Comment:
since you're adding locust CLI, maybe you can add in the `Makefile` a target to run the bench, and something in the README to explain how to run it / when / why ?